README.md

```diff
@@ -65,42 +77,186 @@ This repository has different branches with each branch providing support for up
 * nsxt_ip_blocks
 * nsxt_ip_blocks_facts
 
+#### Policy API
+Policy API modules are aggregated such that logical constructs related to an NSX resource can be configured using a single playbook. They can be identified with prefix *nsxt_policy_*. The below list outlines the supported modules and the resources that can be configured through a module.
 
-# Prerequisites
-We assume that ansible is already installed. 
-These modules support ansible version 2.7 and onwards. 
-
-* PyVmOmi - Python library for vCenter api.
-
-* OVF Tools - Ovftool is used for ovf deployment. 
+1. Tier-0 Gateway (nsxt_policy_tier0)
+   1. Tier-0 Locale Services
+   2. Tier-0 Static Routes
+   3. Tier-0 Interface
+   4. Tier-0 BGP
+   5. Tier-0 BGP Neighbors
+   6. Tier-0 VRF
+   7. Tier-0 BFD Peers
+2. Tier-1 Gateway (nsxt_policy_tier1)
+   1. Tier-1 Locale Services
+   2. Tier-1 Static Routes
+   3. Tier-1 Interface
+3. Segment (nsxt_policy_segment)
+   1. Segment Port
+4. Policy Group (nsxt_policy_group)
+5. Security Policy and Firewall rules (nsxt_policy_security_policy)
+6. IP Pools (nsxt_policy_ip_pool)
+   1. IP Address Pool Block Subnet
+   2. IP Address Pool Static Subnet
+7. IP Blocks (nsxt_policy_ip_block)
+8. BFD Config (nsxt_policy_bfd_config)
+9. VM Tags (nsxt_vm_tags)
 
+Note that to add a new modules in Policy API, it's base class name should be added in the BASE_RESOURCES in module_utils/nsxt_base_resource.py
 
-# Build & Run
+## Build & Run
 
-Install PyVmOmi
+### Install PyVmOmi
 ```
 pip install --upgrade pyvmomi pyvim requests ssl
 ```
-Download and Install Ovf tool 4.3 - [Ovftool](https://my.vmware.com/web/vmware/details?downloadGroup=OVFTOOL430&productId=742)
+### Download and Install Ovf tool 4.3 - [Ovftool](https://my.vmware.com/web/vmware/details?downloadGroup=OVFTOOL430&productId=742)
 (Note: Using ovftool version 4.0/4.1 causes OVA/OVF deployment failure with Error: cURL error: SSL connect error\nCompleted with errors\n)
 
-Download [ansible-for-nsxt](https://github.com/vmware/ansible-for-nsxt/archive/master.zip).
+### Download [ansible-for-nsxt](https://github.com/vmware/ansible-for-nsxt/archive/dev.zip).
```

Comment:
https://github.com/vmware/ansible-for-nsxt/archive/dev.zip
should be https://github.com/vmware/ansible-for-nsxt/archive/master.zip